src/engines/Nybus.Engine.InMemory/InMemory/InMemoryBusEngine.cs

```diff
@@ -128,6 +128,11 @@ public Task NotifyFailAsync(Message message)
 
         public bool IsTypeAccepted(Type type) => _acceptedTypes.Contains(type);
 
+        public Task SendMessageToErrorQueueAsync(Message message)
+        {
+            return Task.CompletedTask;
```

Comment:
Can you try sketch an implementation for this as well?

Reply:
Do you have any suggestion on how best to do that? Do we create a another `ISubject<Envelope>` for the error messages? Or do you have a better idea?

Reply:
Ok, let's take a step back. how users are expected to interact with the DLQ?

Reply:
I would guess in most cases, that would be handled by a different application then, and from the perspective of that application, it would be considered a "normal" queue.

Reply:
uhm... seems correct. Let me sit a bit on this one and see if there is a clean way to implement local DLQ for the in-memory provider.